argo/src/Argo/Socket.hs

```diff
@@ -84,15 +85,15 @@ startListening addr =
 
 -- | Accept a new connection on the given listening socket and
 -- start processing rpc requests.
-acceptClient :: Maybe Handle -> App s -> N.Socket -> IO ()
-acceptClient logH app s =
+acceptClient :: App s -> N.Socket -> IO ()
+acceptClient app s =
 
   do (c, peer) <- N.accept s
      h         <- N.socketToHandle c ReadWriteMode
      -- don't use c after this, it is owned by h
 
      log ("CONNECT: " ++ show peer)
-     forkFinally (serveHandlesNS logH h h app) $ \res ->
+     forkFinally (serveHandlesNS (Just stderr) h h app) $ \res ->
```

Comment:
Do we want to hard-code the use of `stderr` here? For now it seems fine, and perhaps the simplicity of making the decision in this location warrants the (slight) loss of generality as compared to letting the caller specify the handle. What do you think?

Reply:
I thought about it the way you did. I went with simplicity when I couldn't figure out any situation where I'd want something else. If such a situation arises, we can always add the parameter.